docs/mara_lampert/getting_started_with_mambaforge_and_python/readme.md

```diff
@@ -0,0 +1,199 @@
+# Getting started with Mambaforge and Python 
+[Mara Lampert](../readme), January 26th, 2023
+
+## Introduction to Python and Mamba 
+This blog post explains what Python and [Mamba](https://mamba.readthedocs.io/en/latest/installation.html)/ Mambaforge, and how you can download and setup it on your computer. We will also go through some steps how to get started with Bio-image Analysis. 
+
```

Comment:
Maybe a short table of contents? Mac-users could then directly jump to the installation instructions for Mac. You can add links to sections in your file with this syntax:

* [Link text](Link_to_readme#header-with-lowercase-letters-and-no-spaces)

Reply:
In a Jupyter Book there is a table of contents generated automatically

Reply:
No need to do anything